packages/web/app/components/light-wallet-export/LightWalletExportLayoutStepListItem.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from "react";
+
+import * as styles from "./LightWalletExportLayout.module.scss";
+
+type TExternalProps = {
+  title: string;
+  icon: string;
+};
+
+const LightWalletExportLayoutStepListItem: FC<TExternalProps> = ({ title, children, icon }) => (
```

Comment:
```suggestion
const LightWalletExportLayoutStepListItem: React.FunctionComponent<TExternalProps> = ({ title, children, icon }) => (
```

for consistency with rest of the code